crates/primitives/src/transaction/sidecar.rs

```diff
@@ -441,7 +441,7 @@ impl BlobTransactionSidecarRlp {
 #[cfg(any(test, feature = "arbitrary"))]
 impl<'a> arbitrary::Arbitrary<'a> for BlobTransactionSidecar {
     fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
-        let mut arr = [0u8; BYTES_PER_BLOB];
+        let mut arr = vec![0u8; BYTES_PER_BLOB];
```

Comment:
here we're gaining not much from the alloc because in the end we're copying this into an array again, so we should just allow the lint here

Reply:
That sounds fine, I'll make that change. The "fix" here was hacky anyway.

Reply:
Hmm to be clear, I should revert both changes or this one? Both are required.

This linter's limit is 512KB on the stack. This is what I see:
```
error: this function allocates a large amount of stack space
   --> crates/primitives/src/transaction/sidecar.rs:446:18
    |
446 |               .map(|_| {
    |  __________________^
447 | |                 arr = arbitrary::Arbitrary::arbitrary(u).unwrap();
448 | |
449 | |                 // Ensure that each blob is canonical by ensuring each field element contained in
...   |
455 | |                 Blob::from_bytes(arr.as_slice()).unwrap()
456 | |             })
    | |_____________^
    |
    = note: allocating large amounts of stack space can overflow the stack
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_stack_frames
    = note: requested on the command line with `-D clippy::large-stack-frames`
```

Reply:
I think you can allow this with `#[allow(clippy::large_stack_frames)]` and a note

the lint is good, but here we're still moving the value to the stack, so the alloc is kinda pointless